src/main/java/androidx/widget/TextView.kt

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.widget
+
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.DrawableContainer
+import android.support.annotation.DrawableRes
+import android.support.annotation.IntRange
+import android.support.annotation.RequiresApi
+import android.support.v4.content.ContextCompat
+import android.widget.TextView
+
+@DrawableRes
+const val UPDATE_MODE_REMOVE = 0
+@DrawableRes
+const val UPDATE_MODE_KEEP = -1
+
+private val drawableSentinel = DrawableContainer()
```

Comment:
Does this turn into a static field in the bytecode? Or do we need to annotate it as `@JvmField`?